Zend/zend_portability.h

```diff
@@ -435,7 +435,7 @@ char *alloca();
 #ifdef ZEND_WIN32
 #define ZEND_SECURE_ZERO(var, size) RtlSecureZeroMemory((var), (size))
 #else
-#define ZEND_SECURE_ZERO(var, size) memset((var), 0, (size))
+#define ZEND_SECURE_ZERO(var, size) explicit_bzero((var), (size))
```

Comment:
@KalleZ Technical side looks fine to me. I'd only suggest to update these lines where relevant: https://github.com/php/php-src/pull/2163/files#diff-b58ef45738191f784f64ed041461952aL55 It should match the revision we're actually using.